app/style/panel/participants.less

```diff
@@ -207,11 +208,6 @@
     margin: 106px 18px;
     white-space: pre;
   }
-
-  user-list,
-  service-list {
-    margin-top: 16px;
```

Comment:
Where did you move this to?

Reply:
Nowhere. The top margin was there to compensate for the lack of spacing on top, as the content was aligned to the top of the containing div.